gui/wxpython/gui_core/menu.py

```diff
@@ -305,3 +307,37 @@ def OnItemSelected(self, node):
             label = data['description']
 
         self.showNotification.emit(message=label)
+
+
+class FileHistory:
+    """Add recent files history menu"""
+
+    def __init__(self, app_name, parent_menu, pos, history_len=10):
+        self.filehistory = wx.FileHistory(maxFiles=history_len)
+        self.config = wx.Config(app_name, style=wx.CONFIG_USE_LOCAL_FILE)
```

Comment:
Other GRASS GIS things end up in `.grass7` on unix (and something line that on Windows while this uses the wx mechanism. While ideal for wx-only app, GRASS GIS should have this determined independently from wx (e.g., g.extension), so the dir needs to be the GRASS one (the code for it should be somewhere). Well, unless I'm missing something here.